src/components/application_manager/include/application_manager/application_manager_impl.h

```diff
@@ -933,6 +933,7 @@ class ApplicationManagerImpl
       uint32_t connection_key,
       security_manager::SSLContext::HandshakeResult result) OVERRIDE;
 
+  void OnHandshakeFailed() OVERRIDE;
```

Comment:
@AByzhynar I would add description here, What exact Application manager should do OnHandshakeFailed